src/libraries/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/DateTimeHelper.cs

```diff
@@ -92,28 +92,28 @@ private static bool Rfc822DateTimeParser(string dateTimeString, out DateTimeOffs
             DateTimeOffset theTime;
             string[] parseFormat =
             {
-                "ddd, dd MMMM yyyy HH:mm:ss zzz",
-                "dd MMMM yyyy HH:mm:ss zzz",
-                "ddd, dd MMM yyyy HH:mm:ss zzz",
-                "dd MMM yyyy HH:mm:ss zzz",
-
-                "ddd, dd MMMM yyyy HH:mm zzz",
-                "dd MMMM yyyy HH:mm zzz",
-                "ddd, dd MMM yyyy HH:mm zzz",
-                "dd MMM yyyy HH:mm zzz",
```

Comment:
This is removing the overloads without the `:ss` part, why?

Reply:
I'm not sure what you mean; I didn't remove any overloads or change the `:ss` part. All the previously existing formats are still there, just with the correct number of digits for date and year.

Reply:
Ah, sorry, it was hidden under the messages about whitespace error.

Reply:
Oh, that makes sense, there were a LOT of error messages! 😅